package.json

```diff
@@ -35,64 +32,65 @@
   },
   "homepage": "https://github.com/Ivan-Marquez/momentum#readme",
   "standard": {
+    "globals": [
+      "sinon",
+      "expect",
+      "describe",
+      "context",
+      "it",
+      "beforeEach",
+      "assert",
+      "clearModule",
+      "config"
+    ],
     "ignore": [
-      "public/"
+      "src/config/mongoose_seed/"
     ]
   },
-  "apidoc": {
-    "title": "Momentum API Docs",
-    "version": "0.1.0"
-  },
   "engines": {
-    "node": "^6.0"
+    "node": "^8.0"
   },
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
-    "bluebird": "3.4.7",
-    "body-parser": "1.16.1",
-    "bower": "1.8.0",
-    "bunyan": "1.8.5",
+    "bluebird": "3.5.0",
+    "body-parser": "1.17.2",
+    "bunyan": "1.8.10",
     "bunyan-format": "0.2.1",
-    "bunyan-middleware": "0.6.0",
+    "bunyan-middleware": "0.8.0",
     "bunyan-rotating-file-stream": "1.6.2",
-    "cross-env": "3.1.4",
+    "cross-env": "5.0.1",
     "dotenv": "4.0.0",
-    "express": "4.14.1",
+    "express": "4.15.3",
     "express-cluster": "0.0.4",
-    "handlebars-form-helpers": "0.1.3",
-    "hbs": "4.0.1",
-    "helmet": "3.4.0",
+    "helmet": "3.6.1",
     "http-status-codes": "1.1.6",
-    "jsonwebtoken": "7.3.0",
+    "jsonwebtoken": "7.4.1",
     "lodash": "4.17.4",
-    "moment": "2.17.1",
-    "mongodb": "2.2.24",
-    "mongoose": "4.8.4",
-    "node-sass-middleware": "0.11.0",
-    "nodemailer": "3.1.3",
+    "moment": "2.18.1",
+    "mongodb": "2.2.30",
+    "mongoose": "4.11.1",
+    "nodemailer": "4.0.1",
     "passport": "0.3.2",
     "passport-jwt": "2.2.1",
-    "passport-local": "1.0.0",
-    "serve-favicon": "^2.4.0",
-    "slugify": "1.1.0"
```

Comment:
To create [url slugs](https://seo-hacker.com/url-seo-tutorial/). It was being used to retrieve logs by the status field which could contain more than one word.